README.md

```diff
@@ -153,6 +153,37 @@ The latest OMA Lightweight M2M IoT Agent documentation is also available on [Rea
 
 ---
 
+## Roadmap
+
+This product is a FIWARE Generic Enabler. If you would like to learn about the overall Roadmap of FIWARE, please check "Roadmap" on the [FIWARE Catalogue](https://www.fiware.org/developers/catalogue/).
+
+### Introduction
+
+This section elaborates on proposed new features or tasks which are expected to be added to the product in the foreseeable future.  There should be no assumption of a commitment to deliver these features on specific dates or in the order given. The development team will be doing their best to follow the proposed dates and priorities, but please bear in mind that plans to work on a given feature or task may be revised.  All information is provided as general guidelines only,  and this section may be revised to provide newer information at any time.
+
```

Comment:
As any other piece of human generated documentation :) it is easy this section gets out of date. I'd recomment to add this paragraph, as a kind of disclaimer:

```
This section has been last updated in January 2019. Please take into account is content could be obsolete.
```

Reply:
Fixed with 8a9a83d55973af0c0c2548abd615268216c0bbe7